src/openbf.f

```diff
@@ -4,14 +4,14 @@
 C>
 C> @authors J. Woollen, J. Ator,  D. Keyser @date 1994-01-06
 
-C> This subroutine connects a new file to the BUFRLIB software for
+C> Connects a new file to the NCEPLIBS-bufr software for
 C> input or output operations, or initializes the library without
 C> connecting to a file, or changes the verbosity of the library for an
 C> already-connected BUFR file.
 C>
 C> The logical unit numbers LUNIT and LUNDX must already be associated
 C> with actual filenames on the local system, typically via a Fortran "OPEN"
```

Comment:
Remove "typically". Is there actually any other way to associate a logical unit with a file in Fortran?

Reply:
Actually there is.  Some operating systems let you associate a file with Fortran logical unit X by setting FORT_X=file within the UNIX script which calls the code, rather than using an OPEN statement inside the code.

Reply:
And do we support that? (That is, do we test it?)